mindsdb/api/http/namespaces/sql.py

```diff
@@ -112,6 +116,15 @@ def post(self):
             traceback=error_traceback,
         )
 
+        end_time = time.time()
+        log_msg = f"SQL processed in {(end_time - start_time):.2f}s ({end_time:.2f}-{start_time:.2f}), result is {query_response['type']}"
+        if query_response["type"] is SQL_RESPONSE_TYPE.TABLE:
+            log_msg += f" ({len(query_response['data'])} rows), "
+        elif query_response["type"] is SQL_RESPONSE_TYPE.ERROR:
+            log_msg += f" ({query_response['error_message']}), "
+        log_msg += f"used handlers {ctx.used_handlers}"
+        logger.info(log_msg)
```

Comment:
maybe use debug level? `logger.debug`

Reply:
this is info per every sql query, it will be to much of logs in production
also the query itself is logged in debug level
`logger.debug(f"Incoming query: {query}")`